### ISSUE TITLE
Wi-Fi login page broken

### ISSUE THREAD

**Issue by TshuW** [AdguardTeam/AdguardFilters#41779]
***Description***:
Enable AgGuard make Wi-Fi login page broken.
* **Current behaviour**: 

Wi-Fi login page broken with no filters when AgGuard is enabled.

[//]: # (Replace %screenshot_url% below with a link to the screenshot of the problem. Also, you can paste image from clipboard instead. It will be automatically loaded.)
<details><summary>Screenshot:</summary>

<img width="2032" alt="Screen Shot 2019-10-05 at 15 01 05" src="https://user-images.githubusercontent.com/13161779/66251502-4ff99700-e783-11e9-83b2-3b9073b13452.png">

<img width="546" alt="Screen Shot 2019-10-05 at 15 07 48" src="https://user-images.githubusercontent.com/13161779/66251660-378a7c00-e785-11e9-806c-1a114181f664.png">

<img width="1208" alt="Screen Shot 2019-10-05 at 15 03 36" src="https://user-images.githubusercontent.com/13161779/66251663-3e18f380-e785-11e9-912e-97b4fb3fb41a.png">

</details><br/>

* **Expected behaviour**: 

Open Wi-Fi login page successfully.

<details><summary>Screenshot:</summary>

<img width="2032" alt="Screen Shot 2019-10-05 at 15 01 16" src="https://user-images.githubusercontent.com/13161779/66251509-61db3a00-e783-11e9-8ff5-faf8c7699cc1.png">

</details><br/>

***Steps to reproduce the problem***:

1. Enable AdGuard
2. Disable All filters
3. Open Wi-Fi login page

***System configuration***

**Filters:**

None

Information                            | Value
---                                    | ---
Operating system:                      |  macOS
Operating system version (Android/iOS) | 10.14.6
Browser:                               | Safari and Chrome
AdGuard version:                       | 12.1.2.595 release (CL-1.4.138)
Filters enabled:                       | None
AdGuard mode (Android only):           | VPN / Proxy (manual/auto)
Filtering quality (Android only):      | High-quality / High-speed / Simplified
HTTPS filtering (Android only):        | On (Default/Blacklist mode) / Off
Simplified filters (iOS only)          | On / Off
AdGuard DNS:                           | Default?
Stealth mode options (Windows only)    | ?
Helpdesk ID (if exists):               | ?

**Comment by BlazDT** [comment]
Without access to what wi-Fi page not likely possible to fix. Please try to add that IP address to HTTPS exclusions.

**Comment by Alex-302** [comment]
@BlazDT better move to corelibs

**Comment by TshuW** [comment]
@BlazDT This still doesn't work. Is there any way to provide more info? I notice that if I enable AdGuard, many requests to get login page resources doesn't send.

**Comment by Alex-302** [comment]
@TshuW please check with disabled stealth mode and disabled all filters.

**Comment by TshuW** [comment]
@Alex-302 Yes, I tried to disable stealth mode and all filters before, but the issue is still there.

**Comment by Alex-302** [comment]
@TshuW please check this rule in custom filter:
```
@@210.77.16.21$network
```
or
```
@@10.224.20.254$network
```

**Comment by TshuW** [comment]
@Alex-302 The issue is still there.
<img width="1920" alt="Screen Shot 2019-10-10 at 10 36 33 PM" src="https://user-images.githubusercontent.com/13161779/66579028-b846df00-ebae-11e9-9136-d89cc52b83b3.png">
and look like above filters had take effect
<img width="1208" alt="Screen Shot 2019-10-10 at 10 37 46 PM" src="https://user-images.githubusercontent.com/13161779/66579074-cbf24580-ebae-11e9-8bd0-1a1682fa9559.png">

**Comment by Alex-302** [comment]
Strange, `@@10.224.20.254$network` is not triggered.

**Comment by Alex-302** [comment]
Issue moved to [AdguardTeam/CoreLibs #1162](https://github.com/AdguardTeam/CoreLibs/issues/1162) via [**ZenHub**](https://www.zenhub.com/)
